src/theme.js

```diff
@@ -10,7 +10,8 @@ function getTheme({ style, name }) {
 
   const primer = getColors(style);
 
-  const foreground = pick({ light: primer.gray[8], dark: primer.gray[7] });
+  const nonEditorForeground = pick({ light: primer.gray[8], dark: primer.gray[7] });
```

Comment:
Should we call `nonEditorForeground` 👉  `workbenchForeground ` instead? It seems that's how VS Code calls these two areas: https://code.visualstudio.com/docs/getstarted/themes#_customizing-a-color-theme

Reply:
Yeah good suggestion.